ocs_ci/ocs/ui/base_ui.py

```diff
@@ -383,6 +384,7 @@ def __init__(self, driver):
         self.ocp_version = get_ocp_version()
         self.ocp_version_full = version.get_semantic_ocp_version_from_config()
         self.page_nav = locators[self.ocp_version]["page"]
+        self.validation_loc = locators[self.ocp_version]["validation"]
```

Comment:
Please do not add locators to init method.  It will fail if we use OCP4.13 and validation will not exist.